bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/TypeConverter.java

```diff
@@ -268,21 +268,31 @@ public static Set<Type> getConvertibleTypes(Object inputValue, Type targetType,
         int targetTypeTag = targetType.getTag();
         int initialErrorCount = errors.size();
 
+        String errorMessage =
```

Comment:
Shall we rename this to something like `ambiguousTargetErrMsg` and remove unnecessary spaces, if it is better